x-pack/plugins/ml/public/application/components/import_export_jobs/import_jobs_flyout/cannot_read_file_callout.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+
+export const CannotReadFileCallout: FC = () => {
+  return (
+    <>
+      <EuiSpacer size="l" />
+      <EuiCallOut
+        title={i18n.translate('xpack.ml.importExport.importFlyout.cannotReadFileCallout.title', {
+          defaultMessage: 'File cannot be read',
+        })}
+        color="warning"
+      >
+        <FormattedMessage
+          id="xpack.ml.importExport.importFlyout.cannotReadFileCallout.body"
+          defaultMessage="Please select a file contained Machine Learning jobs with have been exported from Kibana using the Export Jobs option"
```

Comment:
Typo - should be `which` instead of `with`

Reply:
https://github.com/elastic/kibana/pull/101037/commits/04c01e9a1778555b0c7480ed2ec46c1f5567a2da